### PR TITLE
_template: add a template for quickly porting exercises to track

### PR DESCRIPTION
**Pull request by sjwarner-bp**
Adds a `_template` directory with a basic exercise outline, to facilitate more uniform contributions.

### CODE REVIEW

File: _template/example.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (budmc29) [inline _template/example.sh]: Can we use `#!/usr/bin/env bash` here?

Comment (sjwarner-bp) [reply _template/example.sh]: Sure thing 🙂 

There are a good few exercises that use the old shebang though - I'll try and work through correcting them, if we think this should be a track-wide change?

**Comment by kotp** [comment]
I think it is good to go!

File: _template/example.sh (hunk 1 above)
Comment (budmc29) [reply _template/example.sh]: Great idea.

**Pull request merged**